src/main/java/org/jenkinsci/plugins/rabbitmqconsumer/channels/AbstractRMQChannel.java

```diff
@@ -79,13 +80,13 @@ public Channel getChannel() {
      *
      * @throws IOException throws if something error.
```

Comment:
I'm no huge fan of redundant and/or obvious Javadoc markup, but technically you should list TimeoutException here too.